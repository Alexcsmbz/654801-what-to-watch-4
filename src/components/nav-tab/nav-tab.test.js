import NavTab from './nav-tab.tsx';

const tab = {
  name: `tab`
};

describe(`NavTab snapshot test`, () => {
  it(`NavTab should render nav tab`, () => {
    const tree = renderer.create(
        <NavTab
          tab={tab}
          onClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
