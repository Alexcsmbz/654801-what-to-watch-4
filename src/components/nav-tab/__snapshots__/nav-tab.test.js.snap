// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavTab snapshot test NavTab should render nav tab 1`] = `
<li
  className="movie-nav__item undefined"
  onClick={[Function]}
>
  <div
    className="movie-nav__link"
  >
    tab
  </div>
</li>
`;
